x-pack/test/observability_functional/apps/observability/pages/alerts/index.ts

```diff
@@ -17,25 +18,98 @@ export default ({ getService, getPageObjects }: FtrProviderContext) => {
   const esArchiver = getService('esArchiver');
   const find = getService('find');
   const { alertControls } = getPageObjects(['alertControls']);
+  const PageObjects = getPageObjects(['home', 'common']);
+  const supertest = getService('supertest');
+  const browser = getService('browser');
+
+  const customThresholdRule = {
+    tags: [],
+    params: {
+      criteria: [
+        {
+          comparator: '>',
+          metrics: [
+            {
+              name: 'A',
+              aggType: 'count',
+            },
+          ],
+          threshold: [1],
+          timeSize: 15,
+          timeUnit: 'm',
+        },
+      ],
+      alertOnNoData: false,
+      alertOnGroupDisappear: false,
+      searchConfiguration: {
+        query: {
+          query: '',
+          language: 'kuery',
+        },
+        index: '90943e30-9a47-11e8-b64d-95841ca0b247',
+      },
+    },
+    schedule: {
+      interval: '1m',
+    },
+    consumer: 'logs',
+    name: 'Custom threshold rule',
+    rule_type_id: 'observability.rules.custom_threshold',
+    actions: [
+      {
+        group: 'custom_threshold.fired',
+        id: 'my-server-log',
+        params: {
+          message:
+            '{{context.reason}}\n\n{{rule.name}} is active.\n\n[View alert details]({{context.alertDetailsUrl}})\n',
+          level: 'info',
+        },
+        frequency: {
+          summary: false,
+          notify_when: RuleNotifyWhen.THROTTLE,
+          throttle: '1m',
+        },
+      },
+    ],
+    alert_delay: {
+      active: 1,
+    },
+  };
 
   describe('Observability alerts >', function () {
     this.tags('includeFirefox');
-
     const testSubjects = getService('testSubjects');
     const retry = getService('retry');
     const observability = getService('observability');
+    let customThresholdRuleId: string;
 
     before(async () => {
       await esArchiver.load('x-pack/test/functional/es_archives/observability/alerts');
       const setup = async () => {
         await observability.alerts.common.setKibanaTimeZoneToUTC();
         await observability.alerts.common.navigateToTimeWithData();
+        await PageObjects.common.navigateToUrl('home', '/tutorial_directory/sampleData', {
+          useActualUrl: true,
+        });
+        await PageObjects.home.addSampleDataSet('logs');
+        const { body: createdRule } = await supertest
+          .post('/api/alerting/rule')
+          .set('kbn-xsrf', 'foo')
+          .send(customThresholdRule)
+          .expect(200);
+
+        customThresholdRuleId = createdRule.id;
       };
       await setup();
     });
 
     after(async () => {
       await esArchiver.unload('x-pack/test/functional/es_archives/observability/alerts');
+      await PageObjects.common.navigateToUrl('home', '/tutorial_directory/sampleData', {
```

Comment:
Do we need to run the flaky test runner?

Reply:
[added](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/8151)